packages/aws-cdk-lib/aws-cloudwatch-actions/lib/lambda.ts

```diff
@@ -0,0 +1,35 @@
+import { Construct } from 'constructs';
+import * as cloudwatch from '../../aws-cloudwatch';
+import * as iam from '../../aws-iam';
+import * as lambda from '../../aws-lambda';
+import { Stack } from '../../core';
+
+/**
+ * Use a Lambda action as an Alarm action
+ */
+export class LambdaAction implements cloudwatch.IAlarmAction {
+  private lambdaFunction: lambda.IAlias | lambda.IVersion | lambda.IFunction
+  constructor(
+    lambdaFunction: lambda.IAlias | lambda.IVersion | lambda.IFunction,
+  ) {
+    this.lambdaFunction = lambdaFunction;
+  }
+
+  /**
+   * Returns an alarm action configuration to use a Lambda action as an alarm action.
+   *
+   * @see https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_PutMetricAlarm.html
+   */
+  bind(_scope: Construct, _alarm: cloudwatch.IAlarm): cloudwatch.AlarmActionConfig {
+    this.lambdaFunction.addPermission('AlarmPermission', {
```

Comment:
Why is the id not generated? Using static value `AlarmPermission` prohibits using the same lambda for multiple actions.

Something like this
```
const lambdaAction = () => new awsCloudWatchActions.LambdaAction(lambda);
alarmConcurrentLambdasMaximumReached.addAlarmAction(lambdaAction());
alarmRdsCpuUtilizationMaximumReached.addAlarmAction(lambdaAction());
```

leads to following error:
```
      throw new Error(`There is already a Construct with name '${childName}' in ${typeName}${name.length > 0 ? ' [' + name + ']' : ''}`);
            ^
Error: There is already a Construct with name 'AlarmPermission' in NodejsFunction [...]
```

Reply:
+1

Reply:
I see. I have submitted a PR.

Reply:
Awesome, thanks for your very fast support!

Reply:
Thanks! If you could upvote in the PR, the review may be prioritized.